.travis.yml

```diff
@@ -12,23 +12,47 @@ env:
     # passphrase for packaging gpg secret key
     - secure: "pZmrxStWj0vXaG/YAyJORvxfaKQHvr+O+66yjFF52gQTVd2zNb83Mn4zDLNqMwU3nkXEKYuFbpXKbDSMX9zNbE7W/w1ARSa8MVXnMVM3jbj+OCaBu7fyVrA0tvT3EZkMh943paTRcSyPSzd61P3CNOkcY1RycDHBbJtNrzSdUbdAy6//TWgNxC66dxSeHASRJIIs49caxQssMMSGRgC5aGH4KLU5eSdP3wnWDEHr+aLqaI2DF2W3lNMI0WJwtGN6cLEYzAsA3BkoPoYRFDd4jjTpolVdrFBXRaT1EIg0FXjsvjyFrQfQLBIqvSx42H5w4rCIYoKYYgRgCKDATLK8XxPbFsgPKLVSHFD6t9ebuYacYxS20oMakkqj5e98qMIWe2vkm+V9YNeADIlhYT+NsF9Kug2B6pLK+U+Hhj6TldFpP18snPXttRKzsg9QrGbnBUVq2wv60U7/3pWs+T3tpr9bIFyaUYbRRAwl/hv2hVkFiSdsiLB6tJphbILB7EDKBCu1iK9PC9tlYoGrqptRxj6bmlawOvcnEKRtoPNJ9eyrvHPDd8FmmW6xWbq89zJ97Ztl7VR8CzNePqvG9vwdtJigXovuXfzu4CHX+jOt3jvllB6rMprsK5r++qBzh1SS24sYqo/APtUfoYdkleWhC+tSuhbucZb2yGUCMo/ZhDo="
   matrix:
-    - TARGET_PLATFORM=el/8
-    - TARGET_PLATFORM=el/7
+    # Packages on packages.microsoft.com
     - TARGET_PLATFORM=el/6
-    - TARGET_PLATFORM=ol/7
-    - TARGET_PLATFORM=ol/6
+    # Removed for now because it has no gcc4.8+ (which is required for security
+    # compliance)
+    # - TARGET_PLATFORM=ol/6
+    - TARGET_PLATFORM=el/7
+    # Removed for now since we don't have oracle linux repos on microsoft
+    # - TARGET_PLATFORM=ol/7
```

Comment:
I really like the comments that state the reason why you intentionally left out some distros

Reply:
Resolved in https://github.com/citusdata/packaging/pull/473